### PR TITLE
Frontend: A smattering of small fixes

### PR DESCRIPTION
**Pull request by danielbeardsley**
Some unused code, asserting some assumptions, adding some test data. See commit descriptions for more info.

### CODE REVIEW

**Comment by ardelato** [comment]
CR 👍

File: models/pull.js [hunk 1]
```diff
@@ -133,12 +130,6 @@ Pull.prototype.getStatus = function getStatus() {
       'commit_statuses' : this.commitStatuses
    };
 
-   status['ready'] =
-            status['dev_block'].length === 0 &&
-            status['deploy_block'].length === 0 &&
-            status['QA'].length >= status['qa_req'] &&
-            status['CR'].length >= status['cr_req'];
```
[the remark this answers is not in the document]
Comment (danielbeardsley) [reply models/pull.js:140]: Yes. The old UI defines it's own ready function:

image

image

**Comment by deltuh-vee** [comment]
Is this repo supposed to be included again? This was on the old UI so it might not be a  big deal if we're switching over soon.
CleanShot 2022-04-05 at 13 22 36

**Comment by danielbeardsley** [comment]
> Is this repo supposed to be included again? This was on the old UI so it might not be a big deal if we're switching over soon.

Not sure I understand. But yes, I'm going to open a pull to delete the old UI very soon, so I'm not worried about it.

**Comment by deltuh-vee** [comment]
QA 🎬 
Looks like some of these changes were already covered by https://github.com/iFixit/pulldasher/pull/274

**Pull request merged**